geomstats/information_geometry/normal.py

```diff
@@ -11,17 +11,19 @@
     PoincareHalfSpace,
     PoincareHalfSpaceMetric,
 )
+from geomstats.information_geometry.information_manifold import InformationManifold
 
 
-class NormalDistributions(PoincareHalfSpace):
+class NormalDistributions(PoincareHalfSpace, InformationManifold):
     """Class for the manifold of univariate normal distributions.
 
     This is upper half-plane.
     """
 
     def __init__(self):
         super().__init__(dim=2)
-        self.metric = FisherRaoMetric()
+        self.shape = (2,)
```

Comment:
Can be deleted (comes for free by initializing `PoincareHalfSpace`).

Reply:
Done.